src/cloud-providers/src/provider.ts

```diff
@@ -133,10 +135,10 @@ export default class Provider {
   /**
    * This method is called to determine whether user already logged in to this provider
    * @public
-   * @returns true if a user already logged in
+   * @returns {Promise<string>} return the access token if a user already logged in
```

Comment:
If we add types in typescript we should not add them to the TSDoc.

Reply:
yeah correct. left over from migrating code from a js source